testsuite/python/CMakeLists.txt

```diff
@@ -196,6 +196,7 @@ python_test(FILE observable_chain.py MAX_NUM_PROC 4)
 python_test(FILE mpiio.py MAX_NUM_PROC 4)
 python_test(FILE gpu_availability.py MAX_NUM_PROC 1 LABELS gpu)
 python_test(FILE features.py MAX_NUM_PROC 1)
+python_test(FILE analyze_distance_empty.py MAX_NUM_PROC 256)
```

Comment:
`MAX_NUM_PROC` should be 1 (just like analyze_distance.py) because the function being tested is not likely to break when more CPU cores are used